assets/tailwind.css

```
@@ -5671,6 +6117,10 @@ html:has(.drawer-toggle:checked) {
   transition-timing-function: cubic-bezier(0, 0, 0.2, 1);
 }
 
+.\[a-zA-Z\:\\-\\\.\] {
+  a-z-a--z: \-\.;
```

Biome (1.9.4)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)